cache/src/commonMain/kotlin/org/mobilenativefoundation/store/cache5/LocalCache.kt

```diff
@@ -1379,6 +1379,29 @@ internal class LocalCache<K : Any, V : Any>(builder: CacheBuilder<K, V>) {
             }*/
         }
 
+        fun activeEntries(): Map<K, V> {
+            if (count.value != 0) { // read-volatile
```

Comment:
Can invert if condition and return eagerly to avoid a nested indentation level 🙏